### PR TITLE
chore: Updating peer library to fix a small issue with relay suspension

### PR DESCRIPTION
**Pull request by pablitar**

### CODE REVIEW

File: kernel/package.json [hunk 1]
```diff
@@ -100,7 +100,7 @@
     "dcl-catalyst-client": "^2.1.5",
     "dcl-crypto": "^2.2.0",
     "dcl-social-client": "^1.3.10",
-    "decentraland-katalyst-peer": "0.2.15",
+    "decentraland-katalyst-peer": "0.2.16",
```
Comment (nchamo) [inline kernel/package.json:103]: Damn, I hate that we still have `katalyst` on some places

**Review by nchamo — APPROVED** [review]

File: kernel/package.json (hunk 1 above)
Comment (pablitar) [reply kernel/package.json:103]: Haha. I don't know who has permissions, but we need to create the `decentraland-catalyst-peer` package on NPM and then we can change the package name in Catalyst.

Comment (nchamo) [reply kernel/package.json:103]: We now have the `@dcl` organization in npm. Maybe we could move the package there? Not necessary for this merge, of course. Just an idea for the future

Comment (pablitar) [reply kernel/package.json:103]: In that case, we could just call this `catalyst-peer` and then we could have this as `@dcl/catalyst-peer`. That would be nice.

Comment (nchamo) [reply kernel/package.json:103]: Could I ask you to create an issue on the catalyst repo? So don't forget 😄

Comment (pablitar) [reply kernel/package.json:103]: https://wiki.uqbar.org/wiki/articles/lombardizacion.html

**Pull request merged**